### DISCUSSION TITLE
multi-select

### DISCUSSION THREAD

**Discussion by balupton** [Tribler/tribler#8359] [Ideas]
- [ ] ability to use cmd+click to additionally select the clicked item (current shift+click behaviour)
- [ ] ability to use shift+click to additionally select all between the last clicked item and the newly clicked item (missing functionality)
- [ ] do not select text when cmd/shift+clicking, the text selection makes it seem that it does not work, should be `cursor: pointer !important; user-select: none;`

in this video, I've applied the stylesheet change, and demonstrating how shift+click doesn't select in-between

https://github.com/user-attachments/assets/1cb47e45-f541-4bcd-990e-02ebd2cf8ca1

**Comment by qstokkink** [comment]
Using shift+click to select multiple torrents and then (1) pause, (2) resume, and (3) export all work for me. Do these features not work for you?

[preview.webm](https://github.com/user-attachments/assets/a3812e17-1c5b-4422-aa14-b408b624e0f4)

**Reply by balupton** [reply]
Yeah don't work in safari nor chrome on macos, tried shift and cmd + click.

**Reply by qstokkink** [reply]
`⌘+click` is not implemented, but `shift+click` should work. I'll try to reproduce this when I have a Mac on hand again to test.

**Reply by balupton** [reply]
tested again, yeah, it just selects all the text between the two clicks

**Reply by balupton** [reply]
ohhhh, okay, maybe it does support multi-click, I just assumed as it selected text, it just didn't work - usually there is a css style for this that disables the cursor change

**Reply by balupton** [reply]
I've updated the OP.

**Reply by qstokkink** [reply]
Cool! That looks much nicer without the text selection.

Thanks for the code. We can implement that.

**Comment by qstokkink** [comment]
I converted this discussion into issue #8370
